lib/coqProject_file.ml

```diff
@@ -101,12 +101,12 @@ and parse_string2 buf s = match Stream.next s with
 | exception Stream.Failure -> raise (Parsing_error "unterminated string")
 
 and parse_skip_comment s = match Stream.next s with
-| '\n' -> ()
+| '\n' | '\r' -> ()
 | _ -> parse_skip_comment s
```

Comment:
Is that correct? If the file contains `\r\n`, then `parse_skip_comment` will stop at `\r` instead of continuing till `\n`.

Reply:
Good point, I dropped this change.  I believe what I had would still work for comments `\r\n` (the `\n` is just white space) but not for, say, `# ... \r blah blah blah\n` (an unlikely case even on Windows).